packages/editor-ui/src/components/InviteUsersModal.vue

```diff
@@ -157,25 +157,25 @@ export default mixins(showMessage).extend({
 				if (invitedEmails.success.length) {
 					this.$showMessage({
 						type: 'success',
-						title: `User${invitedEmails.success.length > 1 ? 's' : ''} invited successfully`,
-						message: `An invite email was sent to ${invitedEmails.success.join(',')}`,
+						title: this.$locale.baseText(invitedEmails.success.length > 1 ? 'USERS_INVITED_SUCCESS' : 'USER_INVITED_SUCCESS'),
```

Comment:
Learned recently that we can use [i18n pluralization](https://kazupon.github.io/vue-i18n/guide/pluralization.html#pluralization). Otherwise I would prefer a single key interpolating the right string segment, instead of two keys that differ in a single char.

Same for others where this happens.

Reply:
needs more thinking to introduce this. will handle this separately because I am not done making changes. there's a second pr